addons/revolutionaries/CfgVehicles_O_Russian.hpp

```diff
@@ -143,6 +143,103 @@ class TACU_Revolutionaries_U_O_Russian_Autorifleman: TACU_Revolutionaries_U_O_Ru
+    backpack = "TACU_Revolutionaries_B_Rocketeer_Black";
+    weapons[] = {"launch_RPG32_green_F","CUP_hgun_Mac10", "Throw", "Put"};
+    respawnWeapons[] = {"launch_RPG32_green_F","CUP_hgun_Mac10", "Throw", "Put"};
+    magazines[] = {mag_4("CUP_30Rnd_45ACP_MAC10_M")};
+    respawnMagazines[] = {mag_4("CUP_30Rnd_45ACP_MAC10_M")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Russian_Saboteur: TACU_Revolutionaries_U_O_Russian_Rifleman01 {
+    displayName = "Saboteur";
+    uniformClass = "CUP_U_C_racketeer_02";
+    Items[] = {mag_5("ACE_fieldDressing"), "ACE_Clacker"};
+    respawnItems[] = {mag_5("ACE_fieldDressing"), "ACE_Clacker"};
+    backpack = "TACU_Revolutionaries_B_Saboteur_Black";
+    weapons[] = {"hgun_PDW2000_F", "Throw", "Put"};
+    respawnWeapons[] = {"hgun_PDW2000_F", "Throw", "Put"};
+    magazines[] = {mag_6("30Rnd_9x21_Mag")};
+    respawnMagazines[] = {mag_6("30Rnd_9x21_Mag")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Russian_Pistoleer01: TACU_Revolutionaries_U_O_Russian_Rifleman01 {
+    displayName = "Pistoleer (Garch)";
+    uniformClass = "CUP_U_C_racketeer_03";
+    linkedItems[] = {DEFAULT_ITEMS, "CUP_V_C_Police_Holster", "G_Balaclava_blk"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "CUP_V_C_Police_Holster", "G_Balaclava_blk"};
+    backpack = "";
+    weapons[] = {"hgun_Rook40_F", "Throw", "Put"};
+    respawnWeapons[] = {"hgun_Rook40_F", "Throw", "Put"};
+    magazines[] = {mag_6("16Rnd_9x21_Mag")};
+    respawnMagazines[] = {mag_6("16Rnd_9x21_Mag")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Russian_Pistoleer02: TACU_Revolutionaries_U_O_Russian_Rifleman01 {
+    displayName = "Pistoleer (FNX)";
+    uniformClass = "CUP_U_C_racketeer_04";
+    backpack = "";
+    weapons[] = {"hgun_Pistol_heavy_01_F", "Throw", "Put"};
+    respawnWeapons[] = {"hgun_Pistol_heavy_01_F", "Throw", "Put"};
+    magazines[] = {mag_6("11Rnd_45ACP_Mag")};
+    respawnMagazines[] = {mag_6("11Rnd_45ACP_Mag")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Russian_Head: TACU_Revolutionaries_U_O_Russian_Rifleman01 {
+    displayName = "Head";
+    uniformClass = "CUP_U_O_CHDKZ_Lopotev";
+    linkedItems[] = {DEFAULT_ITEMS, "V_LegStrapBag_black_F", "CUP_H_PMC_Beanie_Black"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "V_LegStrapBag_black_F", "CUP_H_PMC_Beanie_Black"};
+    backpack = "";
+    weapons[] = {"cup_hgun_Deagle", "Throw", "Put"};
+    respawnWeapons[] = {"cup_hgun_Deagle", "Throw", "Put"};
+    magazines[] = {mag_5("CUP_7Rnd_50AE_Deagle")};
+    respawnMagazines[] = {mag_5("CUP_7Rnd_50AE_Deagle")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
```

Comment:
```suggestion
    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
    headgearList[] = {
        "CUP_H_PMC_Beanie_Black", 1
    };
```
And again, if you don't want facewear, add the CBA_facewearList entry as well, or it will inherit the one from Rifleman01, giving a balaclava.